pkg/apis/componentconfig/types.go

```diff
@@ -149,15 +149,16 @@ type KubeProxyConfiguration struct {
 }
 
 // Currently two modes of proxying are available: 'userspace' (older, stable) or 'iptables'
-// (newer, faster). If blank, use the best-available proxy (currently iptables, but may
-// change in future versions).  If the iptables proxy is selected, regardless of how, but
+// (newer, faster), or 'ipvs'(experimental). If blank, use the best-available proxy (currently iptables,
+// but may change in future versions).  If the iptables proxy is selected, regardless of how, but
 // the system's kernel or iptables versions are insufficient, this always falls back to the
 // userspace proxy.
 type ProxyMode string
 
 const (
 	ProxyModeUserspace ProxyMode = "userspace"
 	ProxyModeIPTables  ProxyMode = "iptables"
+	ProxyModeIPVS      ProxyMode = "ipvs"
```

Comment:
For your reference, #53860 also add IPVS and Kernelspace.

Reply:
Thanks!